static_frame/test/unit/test_frame.py

```diff
@@ -4618,6 +4618,30 @@ def test_frame_from_concat_w(self) -> None:
                 )
 
 
+    def test_frame_from_concat_x(self) -> None:
+        f1 = Frame.from_concat([])
+        f2 = Frame.from_concat([], columns='a')
+        f3 = Frame.from_concat([], index=[])
+        f4 = Frame.from_concat([], name='f4')
+        f5 = Frame.from_concat([], columns='a', index=[], name='f5')
```

Comment:
The diversity of test inputs here is good, but I think this is more maintainable if the Frames are next to the code that tests them, rather then split into two groups.

Reply:
Good call. I will rearrange this code.